noxfile.py

```diff
@@ -1,22 +1,54 @@
 import nox
+from pathlib import Path
 
 nox.options.reuse_existing_virtualenvs = True
 
 
+def _should_install(session):
+    """Decide if we should install an environment or if it already exists.
+
+    This speeds up the local install considerably because building the wheel
+    for this package takes some time.
+
+    We assume that if `sphinx-build` is in the bin/ path, the environment is
+    installed.
+    """
+    bin_files = list(Path(session.bin).glob("*"))
+    sphinx_is_installed = any("sphinx-build" in ii.name for ii in bin_files)
+    force_reinstall = "reinstall" in session.posargs
+    should_install = not sphinx_is_installed or force_reinstall
+    if should_install:
+        session.log("Installing fresh environment...")
+    else:
+        session.log("Skipping environment install...")
+    return should_install
+
+
+@nox.session
+def compile(session):
+    if _should_install(session):
+        session.install(".")
```

Comment:
I think that we do need it, because this is what gives us the theme builder **CLI**, which is not bundle with it by default via `install(".")`. Is that what you were asking?